### ISSUE TITLE
Why MongoDB and not ActiveRecord?

### ISSUE THREAD

**Issue by westonganger** [locomotivecms/engine#1310]
I know this is kind of a silly question. I would like to know the reasoning for why MongoDB/Mongoid was chosen over the usual Rails ActiveRecord?

As a Rails developer analyzing which framework to choose, seeing MongoDB/Mongoid was a bit of a put-off or a red flag, but thankfully its mostly a painless experience.

One issue I do note is that a few times MongoDB has crashed (which also causes Postgres to crash) and must be restarted which is weird coming from a SQL background. I am assuming these crashes happen from a bad wagon deploy.

**Comment by did** [comment]
(duplicate of #1111)

Back in 2009 (so long before Postgresql support of Json column), we chose MongoDB over Postgresql because we had to store different kind of documents (content entries). At this time, it made totally sense.  
Now, if we had to pick a DB, we'd choose PostgreSQL without any hesitation. 

That being said, we've never had any problem with MongoDB in production. No crash, no data loss. It's certainly because we use an external DB provider (https://www.compose.com) but even on our legacy platform, MongoDB has never caused us problems.

Speaking about PostgreSQL, we'd love to do the switch. However it requires a lot of time (at least 2 months of work, including tests, migration procedure, ...etc) and so money :-) And since there is absolutely no business value to it (end users don't care if it runs PG or MongoDB), we've got to look for fundings somewhere else. We're currently thinking about it :-)
